kernel/src/main/scala/tofu/Fire.scala

```diff
@@ -0,0 +1,31 @@
+package tofu
+
+import scala.util.Either
+import tofu.internal.{EffectComp, Effect2Comp}
+import tofu.internal.carriers.FibersCarrier
+
+trait Fire[F[_]] {
+  def fireAndForget[A](fa: F[A]): F[Unit]
+}
+
+object Fire extends EffectComp[Fire] {
+  final implicit def byCarrier[F[_], Fib[_]](implicit
```

Comment:
I think it's better to note somehow that it's just for scala 2.12

Reply:
It's not

Reply:
As in #681 this carrier allows `Fib` to be inferred within the macro, just like shapeless whitebox macros do the typelevel calculations we can depend on types that can be unknown if the needed module isn't in the classpath